config/env_sample.txt

```diff
@@ -1,8 +1,5 @@
-IPE_SHEET_ID=1fvYhtxlu3pa8vNBjOVzp1hwNBsv823mTD3uP6l5_WuI
-IPE_SHEET_NAME=Offering
-
-GSERVICE_ACCOUNT_PATH=secrets/ipe/ipe_gserviceaccount.json
+IPE_SHEET_ID=1UrBaZpR5USS-DbbsdNNzjfEFdLQSnqg5RKUUmHkXHGg
```

Comment:
use a place holder for the GSheet ID

Reply:
I think I resolved this from that I understood you wanted and added some comments around this